template/module/readme/USAGE.md

```diff
@@ -1,11 +1,21 @@
-[ This file must be present and contains the usage instructions
-  for end-users. As all other rst files included in the README,
-  it MUST NOT contain reStructuredText sections
-  only body text (paragraphs, lists, tables, etc). Should you need
-  a more elaborate structure to explain the addon, please create a
-  Sphinx documentation (which may include this file as a "quick start"
-  section). ]
+[ This file is required and contains the instructions on **“how”** to use the module for end-users. 
+
+If the module does not have a visible impact on the user interface, just add the following sentence: 
+
+> This module does not impact the user interface.
+
+If that’s not the case, please make sure that every usage step is covered and remember that images speak more than words!]
 
 To use this module, you need to:
 
-1. Go to ...
+- Go to *App* > Menu > Menu item
+
+  *insert screenshot!*
+
+- In “Contact” form, add a value to field `xyz` > save
```

Comment:
suggestion: imho we should use code markers only when referring to code:

```suggestion
- In “Contact” form, add a value to field *xyz* > save
```

Reply:
@yajo Usually, the field names are formatted in `code` so they are easier to see into the documentation.

Reply:
@TumbaoJu tbh I haven't seen this often in current documentation, is there any guidelines suggesting so?

Reply:
@francesco-ooops : Maybe it is just in Numigi? I always though it was the basic but maybe not.
In my opinion, it is easier to read a documentation that uses the `code` instead of the *italic* format for field names.

Maybe we could make it a new guideline?

What do you think?

Reply:
Here is an example : https://github.com/OCA/web/tree/14.0/web_custom_modifier

Reply:
Seems like field labels are in italics, eg: 

> As system administrator, I go to _Settings / Technical / User Interface / Custom Modifiers_.

While technical names of fields are indeed using `code`

So I would keep italics for label names :)

Reply:
Yes, that's it. IMHO this sentence would be properly formatted: "The field `order_line_id` is labeled *Order Lines*".

Since the documentation is functional, I think most of the times you'll write field labels and not field technical names. That's why I asked for this change.

`web_custom_modifier` is an exception because it requires you to input the technical name of the field. In those cases, code formatting is appropriate.



> Maybe we could make it a new guideline?

Please do! 😊